package.json

```diff
@@ -21,7 +21,8 @@
     "start": "lerna run start",
     "db:create": "lerna run db:create",
     "db:drop": "lerna run db:drop",
-    "hyperchain:configure": "ts-node scripts/setup-hyperchain-config"
+    "hyperchain:configure": "ts-node scripts/setup-hyperchain-config",
+    "basetoken": "ts-node scripts/base-token-data"
```

Comment:
There is no `scripts/base-token-data` in this pull request, is it missing?

Reply:
The script was from the first iteration and not longer exists, I will delete it from the `package.json`.